src/runtime/virtcontainers/container.go

```diff
@@ -43,6 +44,11 @@ func (c *Container) tracingTags() map[string]string {
 	}
 }
 
+const (
+	resourceSwappinessLabel  = "io.katacontainers.resource.swappiness"
+	resourceSwapInBytesLabel = "io.katacontainers.resource.swap_in_bytes"
+)
```

Comment:
Shouldn't these new annotations be defined in `src/runtime/virtcontainers/pkg/annotations/annotations.go`?

Reply:
Moved to annotations.go and rename them to io.katacontainers.container.resource.swappiness and io.katacontainers.container.resource.swap_in_bytes.